modules/client-configuration/nav-client-configuration-guide.adoc

```diff
@@ -40,59 +41,61 @@ endif::[]
 **** xref:contact-methods-rhnsd.adoc[rhnsd]
 ***** xref:contact-methods-osad.adoc[OSAD]
 **** xref:contact-methods-pushssh.adoc[Push via SSH]
+// Registration overviev, clients and different options
+** xref:registration-overview.adoc[Client Registration]
 // SLE Client Registration
-** xref:registration-methods.adoc[Registration Methods]
-*** xref:registration-webui.adoc[Register with the Web UI]
-*** xref:registration-bootstrap.adoc[Register with Bootstrap]
-*** xref:registration-cli.adoc[Register on the Command Line]
+*** xref:registration-methods.adoc[Registration Methods]
+**** xref:registration-webui.adoc[Register with the Web UI]
+**** xref:registration-bootstrap.adoc[Register with Bootstrap]
+**** xref:registration-cli.adoc[Register on the Command Line]
 // SUSE Client Registration
-** xref:registration-overview-suse.adoc[SUSE Client Registration]
-*** xref:clients-sle.adoc[SUSE Linux Enterprise Clients]
-*** xref:clients-opensuse.adoc[openSUSE Clients]
-*** xref:clients-sleses.adoc[Expanded Support Clients]
+*** xref:registration-overview-suse.adoc[SUSE Client Registration]
+**** xref:clients-sle.adoc[SUSE Linux Enterprise Clients]
+**** xref:clients-opensuse.adoc[openSUSE Clients]
```

Comment:
Can we take the opportunity to remove openSUSE and openSUSE MicroOS as SUSE clients, and instead create `openSUSE Clients`, and then have `openSUSE Leap` and `openSUSE MicroOS`?

I only see benefits with this: openSUSE is easier to find and we clearly show what we support (Leap and MicroOS, not tumbleweed, at least not for now).